packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -67,6 +68,14 @@ export class BlockList extends Component {
 		);
 	}
 
+	showGroupAppender() {
```

Comment:
Should we call it group appender? It might be displayed for other types of blocks (media-text for instance).
A more generic `showBlockAppender` could work here, not sure if it's very explicit (maybe `showInlineBlockAppender` or `showFooterBlockAppender`)

Reply:
This Appender button is quite unique because it will be render only inside the InnerBlocks and only for Block that is not empty. That's why I initially name it `groupAppender`.

Maybe more generic name like `shouldShowInnerBlockAppender` will be more suitable ? WDYT @pinarol @tug?

Reply:
`shouldShowInnerBlockAppender` 👍

Reply:
Ok, thanks ! :) apply that in a minute